src/pages-conditional/eth.js

```diff
@@ -144,107 +147,114 @@ const StyledCalloutBanner = styled(CalloutBanner)`
+    description: <Translation id="page-eth-open-desc" />,
   },
   {
     emoji: ":shortcake:",
-    title: "Available in flexible amounts",
-    description:
-      "ETH is divisible up to 18 decimal places so you don't have to buy 1 whole ETH. You can buy fractions at a time – as little as 0.000000000000000001 ETH if you want.",
+    title: <Translation id="page-eth-flexible-amounts" />,
+    description: <Translation id="page-eth-flexible-amounts-desc" />,
   },
 ]
 
 const cardListContent = [
   {
     link: "https://docs.ethhub.io/ethereum-basics/monetary-policy/",
-    title: "Ethereum's monetary policy",
-    description: "EthHub",
+    title: <Translation id="page-eth-monetary-policy" />,
+    description: <Translation id="page-eth-ethhub" />,
     caption: "Updated often",
   },
   {
     link: "https://medium.com/ethhub/why-ether-is-valuable-2b4e39e01eb3",
-    title: "Why Ether is valuable",
-    description: "Anthony Sassano",
+    title: <Translation id="page-eth-value" />,
+    description: <Translation id="page-eth-sassano" />,
     caption: "January 2019",
   },
   {
     link:
       "https://support.mycrypto.com/how-to/getting-started/how-to-buy-ether-with-usd",
-    title: "How to buy Ether",
-    description: "MyCrypto",
+    title: <Translation id="page-eth-how-to-buy" />,
+    description: <Translation id="page-eth-mycrypto" />,
     caption: "Updated often",
   },
 ]
 
 const WhatIsEthereumPage = (props) => {
+  const intl = useIntl()
   const data = props.data
   return (
     <Page>
       <PageMetadata
-        title="What is Ether (ETH)?"
-        description="What you need to know to understand ETH and its place in Ethereum."
+        title={intl.formatMessage({
+          id: "page-eth-whats-eth-meta-title",
+          defaultMessage: getDefaultMessage("page-eth-whats-eth-meta-title"),
+        })}
+        description={intl.formatMessage({
+          id: "page-eth-whats-eth-meta-desc",
+          defaultMessage: getDefaultMessage("page-eth-whats-eth-meta-desc"),
+        })}
         image={data.ogImage.childImageSharp.fixed.src}
       />
       <Content>
         <HeroContainer>
           <Header>
-            <Title>What is Ether (ETH)?</Title>
-            <Slogan>Currency for our digital future</Slogan>
-            <Subtitle>ETH is digital, global money.</Subtitle>
-            <SubtitleTwo>It's the currency of Ethereum apps.</SubtitleTwo>
+            <Title>
+              <Translation id="page-eth-whats-eth" />
+            </Title>
+            <Slogan>
+              <Translation id="page-eth-currency-for-future" />
+            </Slogan>
+            <Subtitle>
+              <Translation id="page-eth-is-money" />
+            </Subtitle>
+            <SubtitleTwo>
+              <Translation id="page-eth-currency-for-apps" />
+            </SubtitleTwo>
             <EthPriceCard />
             <Button to="/get-eth/" title="where to buy eth">
               Get ETH
```

Comment:
This string should be translated (i.e. `<Translation id="page-eth-get-eth-btn" />`).